AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_GPUTest_LightComponent.py

```diff
@@ -0,0 +1,275 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Hydra script that is used to create an entity with a Light component attached.
+It then updates the property values of the Light component and takes a screenshot.
+The screenshot is compared against an expected golden image for test verification.
+
+See the run() function for more in-depth test info.
+"""
+import os
+import sys
+
+import azlmbr.asset as asset
+import azlmbr.bus as bus
+import azlmbr.editor as editor
+import azlmbr.math as math
+import azlmbr.paths
+import azlmbr.legacy.general as general
+
+sys.path.append(os.path.join(azlmbr.paths.devroot, "AutomatedTesting", "Gem", "PythonTests"))
+
+import editor_python_test_tools.hydra_editor_utils as hydra
+from atom_renderer.atom_utils import screenshot_utils
+from atom_renderer.atom_utils import atom_component_helper
+from editor_python_test_tools.editor_test_helper import EditorTestHelper
+
+helper = EditorTestHelper(log_prefix="Atom_EditorTestHelper")
+
+LEVEL_NAME = "auto_test"
+DEGREE_RADIAN_FACTOR = 0.0174533
+
+
+def run():
+    """
+    Sets up the tests by making sure the required level is created & setup correctly.
+    It then executes 2 test cases:
+
+    Test Case - Light Component: Capsule, Spot (disk), and Point (sphere):
```

Comment:
would it be better to move the test case details to the docstring for the function that actually does each test case? run() actually just runs test case area_light() and spot_light(). I'm fine either way, but if a test is added/removed this comment block changes instead of the function comment.

Reply:
splitting it is probably easier to digest